wordpress-comments/src/main/java/org/wordpress/aztec/plugins/wpcomments/toolbar/PageToolbarButton.kt

```diff
@@ -49,7 +51,9 @@ class PageToolbarButton(val visualEditor: AztecText) : IToolbarButton {
     }
 
     override fun inflateButton(parent: ViewGroup) {
-        LayoutInflater.from(context).inflate(R.layout.page_button, parent)
+        val rootView = LayoutInflater.from(context).inflate(R.layout.page_button, parent)
+        val button = rootView.findViewById<ToggleButton>(R.id.format_bar_button_page)
```

Comment:
A quick question @khaykov I was wondering if just putting the logic here is breaking the single responsibility principle. I was gonna put this in another function but it seemed a bit trivial. Nonetheless, let me know so I can adhere to code standards 🙇

Reply:
I see your point, but at the same time, we are kind of limited by past architectural decisions :) In my opinion, `inflateButton` can include the code related to making the button ready for usage (which includes setting up accessibility). We can change the architecture, but it might get tricky from there, and considering our eventual move to block editor, I think this is an acceptable approach.

Reply:
Okay great. Glad we are on the same page 😄